src/Stryker.Core/Stryker.Core/Compiling/CompilingProcess.cs

```diff
@@ -41,77 +42,68 @@ public class CompilingProcess : ICompilingProcess
             _logger = ApplicationLogging.LoggerFactory.CreateLogger<CompilingProcess>();
         }
 
+        private string AssemblyName =>
+            _input.ProjectInfo.ProjectUnderTestAnalyzerResult.AssemblyName;
+
         /// <summary>
         /// The compiling process is closely related to the rollback process. When the initial compilation fails, the rollback process will be executed.
         /// </summary>
-        /// <param name="syntaxTrees">The syntaxtrees to compile</param>
+        /// <param name="syntaxTrees">The syntax trees to compile</param>
         /// <param name="ms">The memory stream to store the compilation result onto</param>
+        /// <param name="symbolStream">The memory stream to store the debug symbol</param>
         /// <param name="devMode"></param>
-        public CompilingProcessResult Compile(IEnumerable<SyntaxTree> syntaxTrees, MemoryStream ms, bool devMode)
+        public CompilingProcessResult Compile(IEnumerable<SyntaxTree> syntaxTrees, Stream ms, Stream symbolStream, bool devMode)
```

Comment:
I think it would be useful to rename ms to something more descriptive to make the differences and relations between ms and symbolStream more clear. Maybe sourceStream or something?

Reply:
renamed to ilstream